src/api/application.md

```diff
@@ -422,7 +422,7 @@ Specifies a check method to recognize native custom elements.
   ```js
   // treat all tags starting with 'ion-' as custom elements
   app.config.compilerOptions.isCustomElement = (tag) => {
-    return tag.startsWith('ion-')
+    return tag.startsWith('icon-')
```

Comment:
Could you please provide more context on why do you think `icon` is better? `ion-` is simply a prefix here, I doubt that `icon-table` would make more sense 🤔

Reply:
I think you are right. Thank you. @skirtles-code